compose.yaml

```diff
@@ -27,51 +28,19 @@ services:
     depends_on:
       - db
       - redis
-      - webpack
     environment:
+      DOCKER: true
+      CHROME_URL: http://browserless:4000
```

Comment:
What are these used for ? Are you trying to use https://github.com/browserless/browserless  to run system spec on docker ?